### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 Results window_Keyboard:When tabbing to table in results section there is an unnecessary focus is moving to the whole section

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17209]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#Keyboard;#AINotInScope;#WCAG2.4.3;#FocusOrder;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. After adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
6. Select top 1000 window will open 
7. .Select top 1000 window will open tab to Results window 
8. Tab to the results section and check if there is an unnecessary focus is moving to the whole page when tabbing to table in results.

## Actual:
When tabbing to table in results section there is an unnecessary focus is moving to the whole section, which will be difficult to a keyboard user and also screen reader is also not announcing any data will confuse the screen reader users.
## Expected:
When tabbing to table in results section there should not be any unnecessary focus is moving to the whole section, instead the focus should move to the table in results section directly.
## User Impact:
Keyboard users will face difficulty in navigating the page and also screen reader users will get confused.
## Attachment
![17209](https://user-images.githubusercontent.com/86601127/149337503-453ce294-d629-4dd2-82a5-afc4b4153f65.gif)

**Comment by smartguest** [comment]
This is likely due to how vscode handles tabbing to a window section, which prioritizes focusing on the main window first as with file explorer, there may need to be some workaround in vscode-mssql or an upstream fix (to allow for a skip into the element inside, or if this is expected with most of vscode, perhaps it should be left as is).

**Comment by Roopesh-T** [comment]
![Presentation1](https://user-images.githubusercontent.com/86601127/178227520-52dfa1dc-c101-44eb-bafc-6523498249bd.gif)
Closing the bug as the above issue is not repro
